chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -301,6 +301,13 @@
         %x{zypper --non-interactive addrepo --refresh #{attrs[:url]} #{name}}
       end
     end
+    # install additional packages
+    os = "#{node[:platform]}-#{node[:platform_version]}"
+    packages = [ ]
+    if node[:provisioner][:packages][os]
+      packages = node[:provisioner][:packages][os]
+    end
+    system("zypper --non-interactive install #{packages.join(' ')}")
```

Comment:
> IMHO it have to complain about missing packages.

Yes, and with the suggestion from @vuntz it would.

Style question: Do we prefer curly braces for one-line blocks? Like:
```ruby
package p { action :install }
```
or even
```ruby
node[:provisioner][:packages][os].each { |p| package p { action :install } }
```